db/range_del_aggregator_bench.cc

```diff
@@ -0,0 +1,197 @@
+//  Copyright (c) 2018-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+
+#ifndef __STDC_FORMAT_MACROS
+#define __STDC_FORMAT_MACROS
```

Comment:
Curious, what does this do?

Reply:
I copied that from another file because I didn't read it carefully and thought it was part of the GFLAGS guard. After looking into it, it seems to be used when including `inttypes.h` in C++ code. I'm not using that header so I removed the `__STD_FORMAT_MACROS` stuff.